WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentBluetoothReaderConnectionAlertsProvider.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+
+struct CardPresentPaymentBluetoothReaderConnectionAlertsProvider: BluetoothReaderConnnectionAlertsProviding {
+    typealias AlertDetails = CardPresentPaymentAlertDetails
+    func scanningForReader(cancel: @escaping () -> Void) -> CardPresentPaymentAlertDetails {
+        .scanningForReaders(endSearch: {})
```

Comment:
❓ do we want to pass `cancel` here?

```suggestion
        .scanningForReaders(endSearch: cancel)
```

Reply:
Good catch!